exoctk/contam_visibility/sossContamFig.py

```diff
@@ -80,256 +304,257 @@ def contam(cube, targetName='noName', paRange=[0, 360], badPA=[], tmpDir="",
-        l = np.array([0.89, 0.99])
-        s4.line(l, y+y1, line_color='black', line_width=1.5)
-        s4.add_layout(Label(x=l.mean(), y=y1, x_units='data', y_units='data',
-                      text='H2O', render_mode='css', text_font_size='8pt'))
-
-        l = np.array([1.09, 1.2])
-        s4.line(l, y+y1, line_color='black', line_width=1.5)
-        s4.add_layout(Label(x=l.mean(), y=y1, x_units='data', y_units='data',
-                      text='H2O', render_mode='css', text_font_size='8pt'))
-
-        l = np.array([1.1, 1.24])
-        s4.line(l, y+y2, line_color='black', line_width=1.5)
-        s4.add_layout(Label(x=l.mean(), y=y2, x_units='data', y_units='data',
-                      text='CH4', render_mode='css', text_font_size='8pt'))
-
-        l = np.array([1.3, lamO2.max()])
-        s4.line(l, y+y1, line_color='black', line_width=1.5)
-        s4.add_layout(Label(x=l.mean(), y=y1, x_units='data', y_units='data',
-                      text='H2O', render_mode='css', text_font_size='8pt'))
-
-        l = np.array([0.7665, 0.7699])
-        s4.line(l[0], [y3, y3+0.02], line_color='black')
-        s4.line(l[1], [y3, y3+0.02], line_color='black')
-        s4.add_layout(Label(x=l.mean(), y=y3+0.02, x_units='data',
-                      y_units='data', text='K', render_mode='css',
-                      text_font_size='8pt'))
-
-        l = np.array([1.1692, 1.1778])
-        s4.line(l[0], [y3, y3+0.02], line_color='black')
-        s4.line(l[1], [y3, y3+0.02], line_color='black')
-        s4.add_layout(Label(x=l.mean(), y=y3+0.02, x_units='data',
-                      y_units='data', text='K', render_mode='css',
-                      text_font_size='8pt'))
-
-        l = np.array([1.2437, 1.2529])
-        s4.line(l[0], [y3, y3+0.02], line_color='black')
-        s4.line(l[1], [y3, y3+0.02], line_color='black')
-        s4.add_layout(Label(x=l.mean(), y=y3+0.02, x_units='data',
-                      y_units='data', text='K', render_mode='css',
-                      text_font_size='8pt'))
-
-        l = np.array([1.1384, 1.1409])
-        s4.line(l[0], [y4, y4+0.02], line_color='black')
-        s4.line(l[1], [y4, y4+0.02], line_color='black')
-        s4.add_layout(Label(x=l.mean(), y=y4+0.02, x_units='data',
-                      y_units='data', text='Na', render_mode='css',
-                      text_font_size='8pt'))
-
-        l = np.array([1.2682])
-        s4.line(l[0], [y4, y4+0.02], line_color='black')
-        s4.add_layout(Label(x=l.mean(), y=y4+0.02, x_units='data',
-                      y_units='data', text='Na', render_mode='css',
-                      text_font_size='8pt'))
-
-        s4.xaxis.major_label_text_font_size = '0pt'
-        s4.yaxis.major_label_text_font_size = '0pt'
-
-        # put all the plots in a grid layout
-        fig = gridplot(children=[[s7, s4, s1, s0], [s6, s5, s2, s3]])
+    TOOLS = 'pan, box_zoom, crosshair, reset, hover, save'

```

Comment:
Looks like you are updating the bokeh plots, did you mean to leave all of the panels below commented out?

Reply:
Yes, these panels are part of the `_original_contam()` function, which is now retired (it was the old code used to calculate just the NIRISS Contamination plots). @mfixstsci

Reply:
We might just want to remove this code as I mentioned to you in person and if we need to reimplements something from it, we can look to previous releases and fold it back in. @bourque do you think this is reasonable?

Reply:
@mfixstsci @jaymedina Yes, I would encourage to remove any unused code as it usually only causes confusion.  As Mees mentions, we can still look back in the version control history if needed.

Reply:
cool sounds good